components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/CommonUtil.java

```diff
@@ -408,4 +414,77 @@ public static void copyDirectory(String sourceDir, String destDir) throws APIImp
             throw new APIImportExportException(errorMessage, e);
         }
     }
+
+
+    /**
+     * Add the type and the version to the artifact file when exporting.
+     *
+     * @param type        Type of the artifact to be exported
+     * @param version     API Manager version
+     * @param jsonElement JSON element to be added as data
+     */
+    public static JsonObject addTypeAndVersionToFile(String type, String version, JsonElement jsonElement) {
```

Comment:
Missing the @return java doc

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/9533/commits/764d4c26fe0a819e718e93d7b165081097a8333e